core/src/main/scala/io/finch/streaming/StreamFromReader.scala

```diff
@@ -0,0 +1,27 @@
+package io.finch.streaming
+
+import com.twitter.concurrent.AsyncStream
+import com.twitter.io.{Buf, Reader}
+
+/**
+  * Create stream S[F, Buf] from com.twitter.io.Reader[Buf]
+  */
+trait StreamFromReader[S[_[_], _], F[_]] {
+
+  def apply(reader: Reader[Buf]): S[F, Buf]
+
+}
+
+object StreamFromReader {
+
+  def instance[S[_[_], _], F[_]](fn: Reader[Buf] => S[F, Buf]): StreamFromReader[S, F] = new StreamFromReader[S, F] {
+    def apply(reader: Reader[Buf]): S[F, Buf] = fn(reader)
+  }
+
+  type AsyncStreamF[F[_], A] = AsyncStream[A]
```

Comment:
Let's just strip out any async-stream APIs - there are better streaming APIs we're providing.

Reply:
I also created a ticket so we kill async-stream all together #1055

Reply:
Removed